ci-operator/jobs/openshift/cluster-api-provider-openstack/openshift-cluster-api-provider-openstack-master-presubmits.yaml

```diff
@@ -65,6 +65,72 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-openstack,?($|\s.*)
+  - agent: kubernetes
+    always_run: false
```

Comment:
done! thanks for the suggestion.

Reply:
this doesn't have to get pulled in with `/test ...`, because it has a `run_if_changed` block.  So whether it is `optional: true` or not is an orthogonal decision based on whether you want it to block when it does run.  E.g. the CVO has a number of jobs that run whenever non-doc code changes which we want to be blockers.